cloud: add the contract tab to billings (#10818)
pingcap/docs: tidb-cloud/tidb-cloud-billing.md

2. Click **Billing**.
3. Click the **Payment Method** tab.
4. Edit the billing profile information, and then click **Save**.

## Contract

> 

Author: ```suggestion
TiDB Cloud provides you with the contract management feature to make contract management easier and more secure. 
```

> If you have agreed with our sales on a subscription and received an email to review and accept the agreement online, you can do the following:

Author: ```suggestion
If you have agreed with our sales on a contract and received an email to review and accept the contract online, you can do the following:
```

2. Click **Billing**.
3. Click the **Payment Method** tab.
4. Edit the billing profile information, and then click **Save**.

## Contract

> 

Reviewer: owner and billing admin can see it

> In the TiDB Cloud console, you can manage your TiDB Cloud subscriptions to meet compliance requirements.

Author: ```suggestion
In the TiDB Cloud console, you can manage your customized TiDB Cloud subscriptions to meet compliance requirements.
```

> TiDB Clould provides the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.

Author: ```suggestion
TiDB Clould provides you with the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.
```

> TiDB Clould provides you with the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.

Author: According to @langyuezhang, this feature acts as an additional customize subscription in addition to the online pricing and CLA, so could we update "manage your customized TiDB Cloud subscriptions"? @langyuezhang PTAL

> TiDB Clould provides you with the online contracts feature to customize TiDB Cloud subscriptions and meet compliance requirements.

Reviewer: 可以的
